sdk/cosmos/azure-cosmos/test/test_crud_subpartition.py

```diff
@@ -0,0 +1,453 @@
+        assert isinstance(created_recorder.body, Mapping)
+        assert 'id' in created_recorder.body
+
+        created_properties = created_collection.read()
+        self.assertEqual('consistent', created_properties['indexingPolicy']['indexingMode'])
+
+        # read collections after creation
+        collections = list(created_db.list_containers())
+        self.assertEqual(len(collections),
+                         before_create_collections_count + 1,
+                         'create should increase the number of collections')
+        # query collections
+        collections = list(created_db.query_containers(
+            {
+                'query': 'SELECT * FROM root r WHERE r.id=@id',
+                'parameters': [
+                    {'name': '@id', 'value': collection_id}
+                ]
+            }))
+
+        self.assertTrue(collections)
+        # delete collection
+        created_db.delete_container(created_collection.id)
+        # read collection after deletion
+        created_container = created_db.get_container_client(created_collection.id)
+        self.__AssertHTTPFailureWithStatus(StatusCodes.NOT_FOUND,
+                                           created_container.read)
+
+        container_proxy = created_db.create_container_if_not_exists(id=created_collection.id,
+                                                                    partition_key=PartitionKey(path=
+                                                                                               ["/id1", "/id2", "/id3"],
+                                                                                               kind='MultiHash'))
+        self.assertEqual(created_collection.id, container_proxy.id)
+        self.assertDictEqual(PartitionKey(path=["/id1", "/id2", "/id3"], kind='MultiHash'),
+                             container_proxy._properties['partitionKey'])
+
+        container_proxy = created_db.create_container_if_not_exists(id=created_collection.id,
+                                                                    partition_key=created_properties['partitionKey'])
+        self.assertEqual(created_container.id, container_proxy.id)
+        self.assertDictEqual(PartitionKey(path=["/id1", "/id2", "/id3"], kind='MultiHash'),
+                             container_proxy._properties['partitionKey'])
+
+        created_db.delete_container(created_collection.id)
+
+    def test_partitioned_collection(self):
+        created_db = self.databaseForTest
+
+        collection_definition = {'id': 'test_partitioned_collection ' + str(uuid.uuid4()),
+                                 'partitionKey':
+                                     {
+                                         'paths': ['/id', '/pk'],
+                                         'kind': documents.PartitionKind.MultiHash,
+                                         'version': 2
+                                     }
+                                 }
```

Comment:
I saw you added samples to the document operations, but let's add container samples showing this as well (how to create subpartitioned containers)

Reply:
There should be samples of creating subpartitioned containers.

Reply:
right - what I meant is we should also have them with the container_management files since those are meant to have those scenarios as well